docs/superpowers/plans/2026-05-29-validation-overridden.part2.md

```
@@ -0,0 +1,1629 @@
+    if got.GateNodeID != "Gate" {
+        t.Errorf("GateNodeID = %q, want Gate", got.GateNodeID)
+    }
+    if len(got.SubgraphPath) != 1 || got.SubgraphPath[0] != "ChildRun" {
+        t.Errorf("SubgraphPath = %v, want [ChildRun]", got.SubgraphPath)
+    }
+}
+```
+
+- [ ] **Step 2: Run, confirm failure**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)